functional-test/src/test/groovy/org/rundeck/util/common/jobs/JobUtils.groovy

```diff
@@ -116,26 +118,38 @@ class JobUtils {
         return jobs[0]
     }
 
+
     static Execution waitForExecutionToBe(
             String state,
             String executionId,
             ObjectMapper mapper,
             RdClient client,
-            int iterationGap,
-            int timeout
+            WaitingTime iterationGap,
+            WaitingTime timeout
+    ) {
+        return waitForExecutionToBe(state, executionId, mapper, client, iterationGap.duration, timeout.duration)
+    }
+
+    static Execution waitForExecutionToBe(
```

Comment:
Should we make this private? If it is too inflexible -- provide a comment saying that the method taking `WaitingTime` should be preferred.

Reply:
My original idea was to just use Duration and delete WaitingTime. I kept at the end it because i found it was nice to have a way to modify a lot of tests waiting time without going all over them. But i still want to provide the option of just providing a Duration instead of having to modify the enum in case some tests need a more fine tuning.